### PR TITLE
linkedin

### PR DESCRIPTION
**Pull request by VibhorCodecianGupta**
Improvements for linkedin

### CODE REVIEW

**Comment by VibhorCodecianGupta** [comment]
@championswimmer #313 

Motley PR https://github.com/coding-blocks/motley/pull/279

File: src/passport/strategies/user/strategy-linkedin.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ module.exports = new LinkedinStrategy({
 
       let profileJson = profile._json
       let oldUser = req.user
+      profileJson.email = profileJson.emails[0].value
```
Comment (championswimmer) [inline src/passport/strategies/user/strategy-linkedin.js]: to the database too

**Comment by VibhorCodecianGupta** [comment]
@championswimmer done

**Comment by VibhorCodecianGupta** [comment]
@championswimmer also please take a look at https://github.com/coding-blocks/judge-workers/pull/20 , been pending for a while now

**Comment by VibhorCodecianGupta** [comment]
@championswimmer done. 
Also please take a look at coding-blocks/judge-workers#20 , been pending for a while now

File: src/db/definitions/social.js [hunk 2]
```diff
@@ -36,6 +36,7 @@ module.exports = {
     linkedin:{
         id: {type: Sequelize.DataTypes.STRING, primaryKey: true},
         username: Sequelize.DataTypes.STRING,
+        email: Sequelize.DataTypes.STRING,
```
Comment (championswimmer) [inline src/db/definitions/social.js]: migrator too

**Comment by VibhorCodecianGupta** [comment]
@championswimmer done!

**Comment by championswimmer** [comment]
>  Also please take a look at coding-blocks/judge-workers#20 , been pending for a while now 


I haven't forgotten about, dont plug issues from other projects here

**Comment by VibhorCodecianGupta** [comment]
@championswimmer resolved conflict here

**Pull request merged**